examples/linear_model/plot_tweedie_regression_insurance_claims.py

```diff
@@ -613,11 +613,11 @@ def score_estimator(
 
 # %%
 #
-# Finally, we can compare the two models using a plot of cumulated claims: for
+# Finally, we can compare the two models using a plot of cumulative claims: for
```

Comment:
if we're replacing this one, we might as well replace other instances. There are variables named cumulated_xxx in the definition of the lorenz_curve for instance

Reply:
I think that comment has been addressed.